lib/features/scan_nfc/cubit/scan_nfc_cubit.dart

```diff
@@ -20,46 +21,56 @@ class ScanNfcCubit extends Cubit<ScanNfcState> {
     ));
   }
 
-  void tagRead() {
+  void stopSimulation() {
```

Comment:
What do mean with stopSimulation?

Reply:
no, you are right, it should be StopScanning